assets/js/modules/analytics-4/utils/account.js

```diff
@@ -67,3 +67,48 @@ export function getAccountDefaults(
 		[ ENHANCED_MEASUREMENT_ENABLED ]: true,
 	};
 }
+
+/**
+ * Appends an account ID to an account object.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} account Account object.
+ * @param {string} idKey   Key of the account ID.
+ * @return {Object} Account object with an appended ID.
+ */
+export const appendAccountID = ( account, idKey = 'account' ) => {
+	const obj = { ...account };
+
+	const matches = account[ idKey ].match( /accounts\/([^/]+)/ );
+	if ( matches ) {
+		obj._id = matches[ 1 ];
+	}
+
+	return obj;
+};
+
+/**
+ * Appends property and account IDs to a property object.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} property Property object.
+ * @param {string} idKey    Key of the property ID.
+ * @return {Object} Property object with appended IDs.
+ */
+export const appendPropertyAndAccountIds = ( property, idKey = 'property' ) => {
```

Comment:
Same as above. I think `populatePropertyWithIDs` similar to what we have now would be a bit more clear. When I first read this I imagined you'd have a `appendPropertyID` function and this was calling them both but that is not the case.